wiki/Ranking_Criteria/Difficulty_Naming/id.md

```diff
@@ -1,6 +1,6 @@
 # Penamaan tingkat kesulitan
 
-Tujuan dari penamaan tingkat kesulitan adalah untuk memberi tahu pemain seberapa sulit map itu. Artikel ini mencantumkan konvensi umum serta alternatif lain yang telah dikumpulkan selama bertahun-tahun. Namun, selama kumpulan dari nama tingkat kesulitan itu bersifat deskriptif dan dengan jelas menunjukkan perkembangan kesulitan, Anda dapat menggunakan jenis skema apapun.
+Tujuan dari penamaan tingkat kesulitan adalah untuk memberi tahu pemain seberapa sulit map itu. Artikel ini mencantumkan konvensi umum serta alternatif lain yang telah dikumpulkan selama bertahun-tahun. Namun, selama kumpulan dari nama tingkat kesulitan itu bersifat deskriptif dan menunjukkan perkembangan kesulitan secara jelas, kamu dapat menggunakan jenis skema apapun.
```

Comment:
```suggestion
Tujuan dari penamaan tingkat kesulitan adalah untuk memberi tahu pemain seberapa sulit map itu. Artikel ini mencantumkan konvensi umum serta alternatif lain yang telah dikumpulkan selama bertahun-tahun. Namun, selama kumpulan dari nama tingkat kesulitan itu bersifat deskriptif dan menunjukkan eskalasi kesulitan secara jelas, kamu dapat menggunakan jenis skema apapun.
```

Reply:
How about `deret`?

Reply:
If it uses `deret` probably most people will thought of `barisan` (or a row of something)  because that is how `deret` is generally used?
while after actually checking the definition of `deret` it actually  makes sense, but i dont think it'll be easily understood by most people

Reply:
ah okay then we can use `eskalasi`